spec/system/partners/family_requests_system_spec.rb

```diff
@@ -10,13 +10,13 @@
   end
 
   describe "for children with different items, from different families" do
-    let(:item_id) { Item.all.sample.id }
+    let(:item_ids) { Item.pluck(:id).sample(2).sort }
```

Comment:
Can we just create new items? I'm not a fan of using auto-generated data for testing. (In other words, auto-generated data is great when you \*don't care\* about the values - but if you're testing that the values are correct, it's much better to set them yourself.)

Reply:
i like the "boy scout rule". But here its relatively complex change because of how `Rspec - FactoryBot - integration tests` setup. It will lead to multiline change across multiple files. Looks like its behind of the scope of current PR. Maybe is better to leave `item_ids` as it is and try to improve tests suite in standalone PR. I guess tests can be speeded up drastically if not auto-generated data...

Reply:
OK... can you explain why?